.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Please use linter for html+css+js from [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js) so that PR shows checks for JavaScript linter rules.

Reply:
This issue is well resolved! ✔️